intake_geopandas/geopandas.py

```diff
@@ -5,14 +5,15 @@
 import geopandas
 from intake.source.base import DataSource, Schema
 
-from . import __version__
+# from . import __version__
```

Comment:
somehow this failed. didnt find the problem and therefore commented out for now

Reply:
the code in `_version` is 2 years old and quite long. do we need to fix this or can we ignore this?

Reply:
trace if I uncomment this line:

```
_________________________________________________________ ERROR collecting tests/test_sql_source.py _________________________________________________________
ImportError while importing test module '/Users/aaron.spring/Coding/intake_geopandas/tests/test_sql_source.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../anaconda3/envs/intake_geopandas-dev/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_sql_source.py:12: in <module>
    from intake_geopandas import SpatiaLiteSource
intake_geopandas/__init__.py:2: in <module>
    import intake
../../anaconda3/envs/intake_geopandas-dev/lib/python3.6/site-packages/intake/__init__.py:31: in <module>
    for name, driver in autodiscover().items():
../../anaconda3/envs/intake_geopandas-dev/lib/python3.6/site-packages/intake/source/discovery.py:145: in autodiscover
    drivers[entrypoint.name] = _load_entrypoint(entrypoint)
../../anaconda3/envs/intake_geopandas-dev/lib/python3.6/site-packages/intake/source/discovery.py:250: in _load_entrypoint
    return entrypoint.load()
../../anaconda3/envs/intake_geopandas-dev/lib/python3.6/site-packages/entrypoints.py:82: in load
    mod = import_module(self.module_name)
../../anaconda3/envs/intake_geopandas-dev/lib/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
intake_geopandas/geopandas.py:8: in <module>
    from . import __version__
E   ImportError: cannot import name '__version__'
```

probably a recent intake change @martindurant ?

Reply:
I doubt it. It looks like a circular import: this line is trying to import from __init__, which has not completed importing yet.  You could use the same code here
```
from ._version import get_versions
__version__ = get_versions()['version']
del get_versions
```

Reply:
thanks